### PR TITLE
Preemption

### PR DESCRIPTION
**Pull request by congfairy**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
> 
> For pull requests on Chinese localization, set Base Branch to release-1.15
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/reference/glossary/host-aliases.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+date: 2019-01-31
+full_link: /docs/reference/glossary/host-aliases.md
+
+short_description: >
+  HostAliases 是要注入 Pod 的 hosts 文件的 IP 地址和主机名之间的映射。
+
+aka:
+tags:
+- operation
+---
```
Comment (tanjunchen) [inline content/zh/docs/reference/glossary/host-aliases.md]: the en file?

File: content/zh/docs/reference/glossary/host-aliases.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+HostAliases 是要注入 Pod 的 hosts 文件的 IP 地址和主机名之间的映射。
+
+<!--more-->
+
+<!--
+[HostAliases](/docs/reference/glossary/host-aliases.md) is an optional list of hostnames and IP addresses that will be injected into the Pod's hosts file if specified. This is only valid for non-hostNetwork Pods.
+-->
+
+[HostAliases](/docs/reference/generated/kubernetes-api/v1.13/#hostalias-v1-corev) 是一个可选的主机名和 IP 地址列表，如果指定，它们将被注入 Pod 的 hosts 文件中。 这仅适用于非 hostNetwork 的 Pod。
```
Comment (tanjunchen) [inline content/zh/docs/reference/glossary/host-aliases.md]: HostAliases  URL has some problems

File: content/zh/docs/reference/glossary/preemption.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+id: preemption
+date: 2019-01-31
+full_link: /docs/concepts/configuration/pod-priority-preemption/#preemption
+short_description: >
+  Kubernetes 中的抢占逻辑通过驱逐该节点上存在的低优先级 Pod 来帮助待处理的 Pod 找到合适的节点。
+
+aka:
+tags:
+- operation
+---
```
Comment (tanjunchen) [inline content/zh/docs/reference/glossary/preemption.md]: 建议留下原文。之间留下一个空行。

File: content/zh/docs/reference/glossary/preemption.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+Kubernetes 中的抢占逻辑通过驱逐该节点上存在的低优先级 Pod 来帮助待处理的 Pod 找到合适的节点。
+
+<!--more-->
+
+<!--
+If a Pod cannot be scheduled, the scheduler tries to [preempt](/docs/concepts/configuration/pod-priority-preemption/#preemption) lower priority Pods to make scheduling of the pending Pod possible.
+-->
+
+
+如果无法调度 Pod，则调度程序会尝试[抢占](/docs/concepts/configuration/pod-priority-preemption/#preemption) 优先级较低的 Pod ，以便可以调度待处理的 Pod 。
```
Comment (tanjunchen) [inline content/zh/docs/reference/glossary/preemption.md]: 链接存在同样的问题。

File: content/zh/docs/reference/glossary/host-aliases.md (hunk 2 above)
Comment (congfairy) [reply content/zh/docs/reference/glossary/host-aliases.md]: could you help me to find the right URL?I am not sure which URL should be used.

Comment (congfairy) [reply content/zh/docs/reference/glossary/host-aliases.md]: @tanjunchen

Comment (tanjunchen) [reply content/zh/docs/reference/glossary/host-aliases.md]: 我建议英文原文是什么url 就 填什么  /cc @markthink

File: content/zh/docs/reference/glossary/host-aliases.md (hunk 1 above)
Comment (tanjunchen) [reply content/zh/docs/reference/glossary/host-aliases.md]: 补充下英文原文

**Pull request merged**